jwala-services/src/main/java/com/cerner/jwala/control/jvm/command/JvmCommandFactory.java

```diff
@@ -203,7 +204,8 @@ private ExecCommand getExecCommandForThreadDump(String scriptName, Jvm jvm) {
                 StringUtils.deleteWhitespace(jvm.getJvmName()) + '/' + jvm.getTomcatMedia().getRootDir())
                 .normalize().toString();
         final DateTimeFormatter formatter = DateTimeFormat.forPattern("yyyyMMdd.HHmmss");
-        final String threadDumpDir = jvm.getTomcatMedia().getRemoteDir().normalize().toString() + "/" + jvm.getJvmName();
+        final String threadDumpDir = Paths.get(jvm.getTomcatMedia().getRemoteDir().toString() + "/../" + "data")
```

Comment:
This path assumes that Jwala is running in a Cerner installed environment, which will most likely not be the case for the rest of the open source community.
Since this path could be anything specified by the user, backing up to a parent directory that may not exist would throw an exception when attempting any kind of operation.

Why not keep it in the JVM directory like before?

Reply:
The jira requests the path of the file to be in data directory